csv_generator/consumers/__init__.py

```diff
@@ -34,7 +37,7 @@ def feed_consumers(file_list: List[str], output_dir: str, create_date: str,
     consumers = _init_consumers(create_date, output_dir, zip_file_name)
 
     for data_file in file_list:
-        print('consuming: ', data_file)
+        LOGGER.debug('consuming: ', data_file)
```

Comment:
The progress doesn't look great if we are printing to the console while it's processing. Probably either always need to log this or ensure it's logged when in batch mode.

Reply:
you may wish to consider 2 logs ... one for auditing purposes that get written to a file and the other more normally to the screen?

Reply:
Not a problem. Printing was just the done in the first instance, once we have some agreed logging requirements they can be easily added.

Reply:
... you could always catch the error and tail the log file ;) ... but there are lots of good options here.

Reply:
I think I was situations like say running it in Jenkins where the machine and log file may no longer exist. The log can be added to that workflow (the customisation) but it might be easier to output "enough" information to the console in that case. (Of course if the exception / error contained all of the information needed that would be preferable)